src/styles/globals.css

```diff
@@ -1,8 +1,11 @@
+@import url('https://db.onlinewebfonts.com/c/bb018e64d01355748d8ddc53553850b9?family=Cerebri+Sans');
+
+@import url('https://fonts.googleapis.com/css2?family=Space+Mono:wght@700&display=swap');
```

Comment:
consider this [article](https://nextjs.org/docs/basic-features/font-optimization), it's the preferred way of adding a font to a nextjs project